README.md

```diff
@@ -1,25 +1,18 @@
 # geodev-hackerlabs
 
-A place to learn how to build geo apps with the ArcGIS Platform.
+> A place to learn how to build geo apps with the ArcGIS Platform.
 
 ![Steps](./geodev-hackerlabs-apps.png)
 
 ## Getting Started
 
-Follow these steps to prepare for the labs.
-
-1. Sign up for an [ArcGIS Developer subscription](https://developers.arcgis.com/en/sign-up/).
-2. Login to [ArcGIS Online](http://arcgis.com).
- * Search for `Geodev Hackerlabs Share` > `Search Groups`.
- * Uncheck `Only search in my account/organization`.
- * Join the group.
-3. Sign up for a GitHub account if you don't have one.
+Sign up for an [ArcGIS Developer subscription](https://developers.arcgis.com/en/sign-up/).
```

Comment:
Add something here to explain why they need the subscription. Plus, they might already have one or a organizational subscription. Needs clarification.